test/bun.js/prøve.test.ts

```diff
@@ -0,0 +1 @@
+console.log("hello non-ascii file path");
```

Comment:
Please use the test runner to run a test so that we have informative output

Avoid using console.log in test output where not strictly necessary